web_src/less/_base.less

```diff
@@ -363,9 +363,15 @@ a.muted:hover,
   background: var(--color-hover);
 }
 
-.ui.menu .ui.dropdown .menu > .selected.item {
-  color: var(--color-text) !important;
-  background: var(--color-hover) !important;
+.ui.menu .ui.dropdown .menu  {
```

Comment:
`stylelint` is complaining about the extra space here. Will fix this later. Waiting for reactions regarding switching to  
![octicon-server.svg](https://user-images.githubusercontent.com/1714243/108667755-248f5d80-74da-11eb-9e8b-569d5c28e223.png) Site Administration